Sources/LocalizedStringKit/LocalizedStringKit.m

```diff
@@ -12,38 +12,59 @@ @interface LocalizedStringKit : NSObject
 
 @implementation LocalizedStringKit
 
-NSString *Localized(NSString *value, NSString *comment) {
-  return [LocalizedStringKit localizeWithValue:value comment:comment keyExtension:nil];
+#pragma mark - Public
+
+NSString *Localized(NSString *_Nonnull value, NSString *_Nonnull comment) {
+  return [LocalizedStringKit localizeWithValue:value comment:comment keyExtension:nil tableName:nil];
+}
+
+NSString *LocalizedWithTable(NSString *_Nonnull value, NSString *_Nonnull comment, NSString *_Nonnull tableName) {
```

Comment:
We could mark this as `NS_REFINED_FOR_SWIFT` if we want to make this cleaner in Swift?

Reply:
Looks like this is not supported for Swift Package Manager. We'll forgo for now.